cypress/e2e/tests/migration/controls/businessservices/create.test.ts

```diff
@@ -91,7 +91,7 @@ describe(["@tier2"], "Bug MTA-2260: Business service validations", () => {
         cy.contains(button, createNewButton).should("exist");
     });
 
-    it("Bug MTA-2260: Business service success alert and unique constraint validation", function () {
+    it("Business service success alert and unique constraint validation", function () {
```

Comment:
I see that you removed the Bug from this TC but the suite (The `describe`) is still marked with the same bug, is that expected?

Reply:
I missed that, thanks.